modules/local/star_genomegenerate.nf

```diff
@@ -28,8 +17,8 @@ process STAR_GENOMEGENERATE {
     path "versions.yml", emit: versions
 
     script:
+    def args   = (task.ext.args ?: '').tokenize()
```

Comment:
What's the point of `tokenize` just to join it again? 
Anyway, doesn't change the functionality....

Reply:
I think this was a weird scenario because we needed to see which parameters were provided and hence split using `tokenize` to see if it is part of the list and then join it back up again to provide to the tool.

Reply:
Two lines below:
`if (args.contains('--genomeSAindexNbases')) {`